app/models/wpcc/percent.rb

```diff
@@ -0,0 +1,9 @@
+module Wpcc
+  class Percent
+    def self.choose_highest(period_percent, user_percent)
```

Comment:
again, I'm going off on one with naming today. sorry! 🙈 
This object doesn't know anything about periods right? it just knows about percents and comparing them. 
We are assuming that we are already in a singular period and all we have is the "legal percent" and the "user input percent" 
Could I suggest we name these variables `legal_percent` along with the `user_percent` ...? wdyt?